src/BlazorWebView/src/Maui/BlazorWebView.cs

```diff
@@ -30,6 +30,11 @@ public BlazorWebView()
 		/// </summary>
 		public string? HostPage { get; set; }
 
+		/// <summary>
+		/// Gets or sets the path for initial navigation within the Blazor navigation context when the Blazor component is finished loading.
+		/// </summary>
+		public string StartPath { get; set; } = "/";
```

Comment:
I think the one thing to check in .NET MAUI is that you can set this property via a XAML binding. Something like this:

```xaml
<BlazorWebView ...... StartPath="{Binding MyStartPathProperty}" />
```

And then have a `MyStartPathProperty` property in the `.xaml.cs` file that has some value (can just be a static value).

I think if that works, this is good to go, and would possibly unlock the "Deep linking" support that Dan mentioned.

Reply:
Thanks! As per our last discussion, I added `BindableProperty` for `StartPath` in `BlazorWebView`. Tested the binding for Windows and Android. It works.